src/engine/LoadingScreen.cs

```diff
@@ -104,6 +125,13 @@ public override void _Ready()
 
         spinner = GetNode<Control>("Spinner");
 
+        randomizeTipTimer = new Timer
+        {
+            Autostart = false, WaitTime = 5, PauseMode = PauseModeEnum.Process, OneShot = false,
+        };
+        randomizeTipTimer.Connect("timeout", this, "RandomizeTip");
+        AddChild(randomizeTipTimer);
```

Comment:
These kind of Godot child nodes should be defined in the LoadingScreen.tscn file.